iphone/Maps/Classes/FirstSession.mm

```diff
@@ -92,6 +101,22 @@ + (BOOL)isFirstSession {
   return [self installationId] != nil && gIsFirstSession;
 }
 
++ (BOOL)isFirstSessionInCurrentVersion {
+  return gIsFirstSessionInCurrentVersion;
```

Comment:
There are several potential flaws with this approach. Let's simplify it for now and always highlight ? and Route To until they are pressed by a user.

Reply:
Tint tab on every opening of PP screen?

Reply:
A user launches the app update or the new install, sees it tinted (blue) and sees a * near ?.
A user presses tinted button Route To (that means that user has discovered how to build routes). Tint disappears forever.
A user presses ? button. * disappears. It is possible that * will appear again next time for another feature/announcement.